lbz/configuration.py

```diff
@@ -0,0 +1,45 @@
+from abc import abstractmethod
+from os import getenv
+from typing import Any, Callable, Generic, Optional, TypeVar
+
+from lbz.aws_boto3 import SSM
+
+T = TypeVar("T")
+
+
+class BaseValue(Generic[T]):
+    def __init__(
+        self,
+        key: str,
```

Comment:
Maybe `name`? Key is strongly related to dictionaries and env variables 🤔 

But it is only a suggestion.

Reply:
that the point its the key to get the value

Reply:
On the other hand, we have `SSMValue` which operates with the name of the parameter 😉 

But I won't be complaining about it - the key is ok enough for now 👍